trade-finance-manager-ui/templates/case/amendments/_macros/amendment-what-is-a-bank-review-date.njk

```diff
@@ -7,7 +7,7 @@
     summaryText: "What is a bank review date",
     text: "The bank review date is when you decide in accordance with your usual policies and procedures for such facilities whether to continue or terminate the facility based on the borrower's needs and circumstances.",
     attributes: {
-      'data-cy': 'bank-review-date-details'
+      'data-cy': 'amendment--bank-review-date-details'
```

Comment:
remove extra -

Reply:
This is in keeping with the existing convention - all amendment `data-cy`s are of the format `amendment--{amendment feature}`